src/bases/createWidgetBase.ts

```diff
@@ -117,26 +118,21 @@ const createWidget: WidgetFactory = createStateful
 				const props: VNodeProperties = {};
 
 				this.nodeAttributes.forEach((fn) => {
-					const newProps: VNodeProperties = fn.call(this, assign({}, props));
+					const newProps: VNodeProperties = fn.call(this);
```

Comment:
Ah, my other comment was the other direction... I guess would we see a situation where the current set of properties would effect the return?  I could see something like that, though obviously it would create dependencies between the functions which means potentially surprising results.  This would mean though that the function would be going in "blind" and have no idea that it might be overriding a value.